features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/ImageWireframeHelper.kt

```diff
@@ -65,7 +76,179 @@ internal class ImageWireframeHelper(
+                        index = ++currentIndex,
+                        x = drawableX,
+                        y = drawableY,
+                        width = drawable.intrinsicWidth
+                            .densityNormalized(density).toLong(),
+                        height = drawable.intrinsicHeight
+                            .densityNormalized(density).toLong(),
+                        drawable = drawable,
+                        shapeStyle = null,
+                        border = null
+                    )?.let { resultWireframe ->
+                        result.add(resultWireframe)
+                    }
+                }
+            }
+        }
+
+        return result
+    }
+
+    private fun createCompoundDrawableLocationMap(
+        view: TextView,
+        drawable: Drawable,
+        density: Float,
+        index: Int
+    ): HashMap<Int, Pair<Long, Long>> {
+        val compoundDrawableLocationMap = HashMap<Int, Pair<Long, Long>>()
+
+        val globalBounds = viewUtilsInternal.resolveDrawableBounds(
+            view,
+            drawable,
+            density
+        )
+
+        val drawableWidth = drawable.intrinsicWidth
+            .densityNormalized(density)
+        val drawableHeight = drawable.intrinsicHeight
+            .densityNormalized(density)
+        val viewHeight = view.height
+            .densityNormalized(density)
+        val viewWidth = view.width
+            .densityNormalized(density)
+        val viewPaddingStart = view.paddingStart
+            .densityNormalized(density)
+        val viewPaddingTop = view.paddingTop
+            .densityNormalized(density)
+        val viewPaddingBottom = view.paddingBottom
+            .densityNormalized(density)
+        val viewPaddingEnd = view.paddingEnd
+            .densityNormalized(density)
+
+        @Suppress("MagicNumber")
+        compoundDrawableLocationMap[index] = when (index) {
+            0 -> getCompoundDrawableStartCoords(
+                viewPaddingStart,
+                viewHeight,
+                drawableHeight,
+                globalBounds
+            )
+            1 -> getCompoundDrawableTopCoords(
+                viewPaddingTop,
+                viewWidth,
+                drawableWidth,
+                globalBounds
+            )
+            2 -> getCompoundDrawableEndCoords(
+                viewPaddingEnd,
+                viewWidth,
+                viewHeight,
+                drawableWidth,
+                drawableHeight,
+                globalBounds
+            )
+            3 -> getCompoundDrawableBottomCoords(
+                viewPaddingBottom,
+                viewWidth,
+                viewHeight,
+                drawableWidth,
+                drawableHeight,
+                globalBounds
+            )
+            else -> error("Unexpected drawable index: $index")
+        }
+
+        return compoundDrawableLocationMap
+    }
+    private fun getCompoundDrawableStartCoords(
```

Comment:
what are the start/top/end/bottom coordinates of the compound drawable? what do they describe?

Reply:
It describes the absolute positions of the drawables surrounding the `TextView`